CoffeeShop/src/main/java/client/view/OrderView.java

```diff
@@ -0,0 +1,85 @@
+package client.view;
+
+import client.controller.OrderController;
+import exceptions.InvalidOrderException;
+import interfaces.Observer;
+import order.Order;
+import javax.swing.*;
+import java.awt.*;
+import java.util.Queue;
+import java.util.UUID;
+
+public class OrderView extends JFrame implements Observer {
+    private OrderController controller;
+    private JTextArea orderDisplay;
+    private JButton placeOrderButton;
+    private JButton removeOrderButton;
+
+    public OrderView(OrderController controller) {
+        this.controller = controller;
+        setupUI();
+    }
+
+    private void setupUI() {
+        this.setTitle("Coffee Shop Orders");
+        this.setSize(400, 300);
+        this.setDefaultCloseOperation(JFrame.EXIT_ON_CLOSE);
+        this.setLayout(new BorderLayout());
+
+        orderDisplay = new JTextArea();
+        orderDisplay.setEditable(false);
+        this.add(new JScrollPane(orderDisplay), BorderLayout.CENTER);
+
+        JPanel buttonPanel = new JPanel();
+        placeOrderButton = new JButton("Place Order");
+        removeOrderButton = new JButton("Remove Order");
+        buttonPanel.add(placeOrderButton);
+        buttonPanel.add(removeOrderButton);
+        this.add(buttonPanel, BorderLayout.SOUTH);
+
+        // Add button listeners
+        placeOrderButton.addActionListener(e -> {
+            Order newOrder = null;  // You need to get details from the user
+            try {
+                newOrder = new Order();
+            } catch (InvalidOrderException ex) {
+                throw new RuntimeException(ex);
+            }
+            controller.placeOrder(newOrder);
+        });
+
+        removeOrderButton.addActionListener(e -> {
+            String orderIDStr = JOptionPane.showInputDialog("Enter Order ID to remove:");
+            if (orderIDStr != null) {
+                try {
+                    UUID orderID = UUID.fromString(orderIDStr);
+                    controller.removeOrder(orderID);
+                } catch (IllegalArgumentException ex) {
+                    showErrorMessage("Invalid Order ID format!");
+                }
+            }
+        });
```

Comment:
```suggestion
            if (orderIDStr == null) {
                 showErrorMessage("Invalid Order ID format!");
                 return;
             }
             
             controller.removeOrder(UUID.fromString(orderIDStr));
        });
```

This way we exit early if the orderIdString is null